CourseStream/CourseStream/Program.cs

```diff
@@ -112,5 +112,21 @@ static async Task<object> AddCourse([FromServices] CourseStreamService service,
 	return new { Message = "Course added successfully." };
 }
 
+async Task<IActionResult> EnrollStudent([FromServices] CourseStreamService service, int studentId, int courseStreamId)
+{
+	try
+	{
+		await service.EnrollStudentAsync(studentId, courseStreamId);
+		return new OkObjectResult(new { Message = "Student enrolled successfully." });
+	}
+	catch (ConflictException ex)
+	{
+		return new ConflictObjectResult(new { ErrorMessage = ex.Message });
+	}
+	catch (Exception ex)
+	{
+		return new BadRequestObjectResult(new { ErrorMessage = ex.Message });
+	}
```

Comment:
Let's handle this with `ExceptionHandlingFilter`

Reply:
Handled exceptions